.openshift-ci/build/generate-db-dump.sh

```diff
@@ -12,10 +12,10 @@ gate_job generate-db-dump
 generate_db_dump() {
     info "Generating DB dump"
 
-    groupadd -g 71 pg
-    adduser pg -u 71 -g 71 -d /var/lib/postgresql -s /bin/sh
+    groupadd -g 1001 pg
+    adduser pg -u 1001 -g 1001 -d /var/lib/postgresql -s /bin/sh
 
-    runuser -l pg -c "$ROOT/scripts/ci/postgres.sh start_postgres"
+    runuser -l pg -c "PATH=$PATH:/usr/pgsql-$PG_MAJOR/bin/ $ROOT/scripts/ci/postgres.sh start_postgres"
```

Comment:
I actually fixed the `PATH` [when updating the base test CI image](https://github.com/stackrox/rox-ci-image/commit/da336c78463a6597a938d2b781d7b82dfff42c33#diff-f849cde71446382ccc46e79ca6640ed1aec3c28388622eb99e61e339c7a70a91R38). I think what we need to do is to update the `FROM` from the `.openshift-ci/Dockerfile*`. Does that make sense?

The changes for higher group/user IDs is OK. Although, why don't we simply use the next available ID or use [`-r` (`--system`)](https://man7.org/linux/man-pages/man8/groupadd.8.html)?

Reply:
Humm, I actually checked the build log one more time, and I noticed that [PATH was correctly set to include `/usr/pgsql-12/bin`](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-stackrox-scanner-master-merge-e2e-tests/1559333718210908160#1:build-log.txt%3A124).

Reply:
I think we should look into using `--preserve-environment` when using `runserver`, so that the environment is not lost in the subprocess (in this case, the shell running `postgres.sh`).

Reply:
Argh, `--preserve-environment` is ignored when using `--login`, as per [the man page](https://manpages.debian.org/testing/util-linux/runuser.1.en.html).

Reply:
I guess then just passing `PATH=$PATH` would do the trick. Adding a comment on why would be nice too.